tests/framework/utils_vsock.py

```diff
@@ -180,7 +180,7 @@ def check_guest_connections(vm, server_port_path, blob_path, blob_hash):
     cmd += "  ({})& ".format(worker_cmd)
     cmd += '  workers="$workers $!";'
     cmd += "done;"
-    cmd += "for w in $workers; do wait $w || exit 1; done"
+    cmd += "for w in $workers; do wait $w || (wait; exit 1); done"
```

Comment:
could we just replace this with a single `wait || exit 1`

Reply:
Actually it looks like `wait` just returns 0 with no indication if any failed.

An alternative may be https://unix.stackexchange.com/questions/742036/using-wait-bash-posix-and-fail-if-one-process-fails-in-a-script

Reply:
Yeah, `wait` on its own just returns the status code of the last child process that exits, or sth like that. 

> An alternative may be https://unix.stackexchange.com/questions/742036/using-wait-bash-posix-and-fail-if-one-process-fails-in-a-script